code/modules/reagents/reagent_dispenser.dm

```diff
@@ -412,17 +412,28 @@
 	desc = "Originally used to store liquids & powder. It is now used as a source of comfort. This one is made of metal."
 	layer = TABLE_LAYER
 	flags = FPRINT | TWOHANDABLE | MUSTTWOHAND // If I end up being coherent enough to make it holdable in-hand
-	throwforce = 40 // Ends up dealing 20~ brute when thrown because thank you, based throw damage formula
```

Comment:
sneaky

Reply:
most likely just for parity with #28628

Reply:
So leave it to that PR
You don't need parity

Reply:
Yes dad 😔 

I shall prepare for unforeseen consequences.